main.go

```diff
@@ -204,6 +228,15 @@ func (svr *bridge) handle_call(w http.ResponseWriter, r *http.Request) {
 				}
 			}
 
+			if *details == 1 {
+				if strings.HasPrefix(alert.GeneratorURL, "http") {
```

Comment:
I really like the idea of making it clear which source sent the alert. It occurs to me that we could also always set the `client::notification.click` action to be the GeneratorURL if it's set (regardless of detail level). This could be nice and handy to allow a user to shortcut straight to alertmanager from a notification. WDYT of that idea?

Reply:
Added it but unfortunately I can not test that feature as I am using PushBits, instead of Gotify, that does not support that for now.